src/server/lockers/reservations/ConfigVars.ts

```diff
@@ -0,0 +1,54 @@
+export const lockerReservationIncluder = {
+    LockerReservation: {
+        where: {
+            active: true
+        },
+        select: {
+            id: true,
+            endDate: true,
+            group: {
+                select: {
+                    id: true,
+                    groupType: true,
+                    class: {
+                        select: {
+                            year: true
+                        }
+                    },
+                    committee: {
+                        select: {
+                            name: true
+                        }
+                    },
+                    interestGroup: {
+                        select: {
+                            name: true
+                        }
+                    },
+                    manualGroup: {
+                        select: {
+                            name: true
+                        }
+                    },
+                    omegaMembershipGroup: {
+                        select: {
+                            omegaMembershipLevel: true
+                        }
+                    },
+                    studyProgramme: {
+                        select: {
+                            name: true
+                        }
+                    }
```

Comment:
This part of the selector should be refactored with visibility similar thing has been done there to infer the group name